posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: "Meet our open-source champions: Billy Dawson"
```

Comment:
need to remove colons from the metadata field values- they cause the doc to fail. You can replace with a dash instead